django_rq/views.py

```diff
@@ -58,6 +60,12 @@ def stats(request):
     return render(request, 'django_rq/stats.html', context_data)
 
 
+def _get_all_workers(config, workers_collections):
+    for collection in workers_collections:
```

Comment:
I call it on `workers.py`, inside `collect_workers_by_connection`. I thought would be better.
```python
workers_collections = []
for item in queue_list:
    connection_params = filter_connection_params(item['connection_config'])
# ...
```